test/support/factories/alerts/alert.ex

```diff
@@ -6,8 +6,11 @@ defmodule Test.Support.Factories.Alerts.Alert do
   use ExMachina
 
   alias Alerts.{Alert, Priority}
+  alias Test.Support.Factories.Alerts.InformedEntity
   alias Test.Support.Factories.Alerts.InformedEntitySet
 
+  @high_priority_effects [:delay, :shuttle, :station_closure, :suspension]
```

Comment:
It'd be nice to not have to duplicate this from its source of truth.

Reply:
Yeah.... What IS its source of truth?

Also I just realized this point might be moot, since `SystemStatus.Subway.subway_status/2` doesn't actually do anything with the `effect` now. 🤔

Reply:
Wait, what is giving the subway status feature the "high priority" alerts then?? 😅

Reply:
In `alerts.ex`, the `filter_relevant/1` function [ensures that only certain alerts are returned](https://github.com/mbta/dotcom/blob/fb5974c36f0aae16b5a4e0a6abb78aa6ccbe4563/lib/dotcom/system_status/alerts.ex#L88-L90), and which effects are "relevant" is defined [here](https://github.com/mbta/dotcom/blob/fb5974c36f0aae16b5a4e0a6abb78aa6ccbe4563/lib/dotcom/system_status/alerts.ex#L7). 

In this PR, the results from `alerts.ex` are passed into `subway_status/2`.

That's how it works now - I'm not overly attached to that thought - there's probably a better way.

Reply:
Ah okay! In that case, the link behind "here" is the source of truth I'm referring to. 

If there were a function there that exposes that list, say `Dotcom.SystemStatus.Alerts.relevant_effects()`, you could call that in this module where needed.

Reply:
This is gone now 👋